src/components/ha-tabs.ts

```diff
@@ -34,14 +34,17 @@ export class HaTabs extends PaperTabs {
 
       superStyle!.appendChild(
         document.createTextNode(`
+          :host {
+            padding-top: .5px;
+          }
           .not-visible {
             display: none;
           }
-          :host > paper-icon-button:first-of-type {
-            padding-left: 0;
-          }
           paper-icon-button {
-            margin: 0 -8px 0 0;
+            width: 24px;
+            height: 48px;
```

Comment:
Why are we introducing a `width` and `height` for paper-icon-button?

Reply:
This is strictly for the scroll arrows in paper-tabs to allow for more space for tabs.